### DISCUSSION TITLE
Welcome to C-Blosc2 Discussions!

### DISCUSSION THREAD

**Discussion by FrancescAlted** [Blosc/c-blosc2#582] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EmilDohne** [comment]
I'm a Pipeline TD working in the VFX industry and am currently working on an API to read/write Photoshop files and use c-blosc2 for in-memory compression of the image data and it is genuinely incredible how it is faster and smaller compared to copying the data around. Hats off to you!
